### PR TITLE
Adding video capabilities to fluid 250 ad banner

### PR DESCRIPTION
**Pull request by austinh7**
This adds the ability to have video from YouTube or Vimeo embedded into the fluid 250 ad format.

Extra fields have been added to the Fluid 250 DFP template. (See export of template here -> https://github.com/guardian/commercial-responsive-ads/blob/master/DFP__export-templates/Fluid%20250%20V2.txt)

These give the ability to position and size the video as you want to.

screen shot 2015-01-08 at 15 44 45

### CODE REVIEW

File: static/src/javascripts/projects/common/views/commercial/creatives/fluid250.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     ">
     <a href="{{clickMacro}}{{link}}" target="_blank">
         <div class="gs-container">
+            {{videoD}}
```
Comment (ironsidevsquincy) [inline static/src/javascripts/projects/common/views/commercial/creatives/fluid250.html]: what's the `D` for? isn't `video` enough?

File: static/src/javascripts/projects/common/modules/commercial/creatives/fluid250.js [hunk 2]
```diff
@@ -23,15 +23,41 @@ define([
...
+        var templateOptions, leftMargin, leftPosition, rightPosition, topMargin, videoDesktop;
+
+        templateOptions = {
             showLabel: (this.params.showAdLabel === 'hide') ?
                 'creative__label--hidden' : ''
         };
-        $.create(template(fluid250Tpl, merge(this.params, templateOptions))).appendTo(this.$adSlot);
+
+        leftMargin = (this.params.videoPositionH === 'center' ?
+            -(this.params.videoWidth / 2) : 0
```
Comment (ironsidevsquincy) [inline static/src/javascripts/projects/common/modules/commercial/creatives/fluid250.js]: is it necessary to allow them to configure the size? could just say it has to be a defined size, would safe on setting values in dfp, calculating margins, etc

**Comment by ironsidevsquincy** [comment]
wonder if we should create a new template for video fluids?

File: static/src/javascripts/projects/common/views/commercial/creatives/fluid250.html (hunk 1 above)
Comment (austinh7) [inline static/src/javascripts/projects/common/views/commercial/creatives/fluid250.html]: Good spot, D was there when I was going to be using video in mobile version as well which we aren't doing any more.

File: static/src/javascripts/projects/common/modules/commercial/creatives/fluid250.js (hunk 2 above)
Comment (austinh7) [inline static/src/javascripts/projects/common/modules/commercial/creatives/fluid250.js]: We need size as options are needed if they want to go full bleed, (ie 250px high) or leave some space around so the video integrates into the ad a bit more.

**Comment by Calanthe** [comment]
:+1:

**Pull request merged**